src/bitwarden_exporter/bw_list_process.py

```
@@ -156,51 +119,55 @@ def process_list(settings: BitwardenExportSettings) -> None:
+        bw_folder = BwFolder(**bw_folder_dict)
+        if not bw_folder.id:
+            continue
+
+        bw_process_items.folders[bw_folder.id] = bw_folder
+
     LOGGER.warning("Fetching summary: application retrieved folders from Bitwarden CLI")
-    LOGGER.info("Total Folders Fetched: %s", len(bw_folders))
+    LOGGER.info("Total Folders Fetched: %s", len(bw_process_items.folders))
 
-    no_folder_items: List[BwItem] = []
+    bw_organizations_dict = json.loads((bw_exec(["list", "organizations"], is_raw=False)))
```

Ruff (0.14.2)

Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)